cmd/maya-apiserver/cstor-operator/spc/storagepool_create.go

```diff
@@ -134,6 +135,12 @@ func (cb *CasPoolBuilder) withAnnotations(annotations map[string]string) *CasPoo
 	return cb
 }
 
+// WithPoolROThreshold set PoolROThreshold value
+func (cb *CasPoolBuilder) WithPoolROThreshold(poolROThreshold int) *CasPoolBuilder {
```

Comment:
@kmova Should we add default value for pool readonly threshold, like 80, so if pool storage gets consumed up-to 80% then replica controller will set readonly mode to pool and no IOs will be allowed(volume creation will be alloed) on the pool unless user expand the pool.?